packages/cli/src/util/serveUtil.js

```diff
@@ -26,7 +26,7 @@ const addHandler = (site, onePagePath) => (filePath) => {
   }
   Promise.resolve('').then(async () => {
     if (site.isFilepathAPage(filePath) || site.isDependencyOfPage(filePath)) {
-      return site.rebuildSourceFiles(filePath);
```

Comment:
hmm, maybe there's something wrong here. I'm not sure why we are rebuilding all source files when one file is added/removed. Anyhow, it's not related to this PR.